pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -113,8 +118,10 @@ async function convertAndParsePlist(inputPath: string): Promise<undefined|Recurs
   try {
     plutilResult = await spawnFile('plutil', args, { stdio: [body, 'pipe', 'pipe'] });
   } catch (error: any) {
-    console.log(`Error parsing deployment profile plist file ${ inputPath }\n${ error }`);
-    throw new DeploymentProfileError(`Error loading plist file ${ inputPath }: ${ getErrorString(error) }`);
+    console.log(`Error parsing deployment profile plist file ${ inputPath }\n${ error }`, error);
```

Comment:
Do we need to say `error` twice here?
(I added a `debugE` to `Log`; that might be closer to what you want. Might need a `logE` though.)

Reply:
ok....